.github/workflows/fuzz.yml

```diff
@@ -13,6 +13,8 @@ on:
     branches:
     - master
 
+permissions: {}
```

Comment:
Please put in the same order on the job files, between concurrency and jobs sections, thanks.